ui/tests/acceptance/client-detail-test.js

```diff
@@ -1140,12 +1140,21 @@ module('Acceptance | client detail', function (hooks) {
   testFacet('Status', {
     facet: ClientDetail.facets.status,
     paramName: 'status',
-    expectedOptions: ['Pending', 'Running', 'Complete', 'Failed', 'Lost'],
+    expectedOptions: [
+      'Pending',
+      'Running',
+      'Complete',
+      'Failed',
+      'Lost',
+      'Unknown',
+    ],
     async beforeEach() {
       server.createList('job', 5, { createAllocations: false });
-      ['pending', 'running', 'complete', 'failed', 'lost'].forEach((s) => {
-        server.createList('allocation', 5, { clientStatus: s });
-      });
+      ['pending', 'running', 'complete', 'failed', 'lost', 'unknown'].forEach(
+        (s) => {
+          server.createList('allocation', 5, { clientStatus: s });
+        }
+      );
 
       await ClientDetail.visit({ id: node.id });
```

Comment:
**question:** Does the Nomad API change the state of the node to `disconnected` instead of marking the node as `down`? cc:  @DerekStrickland 

If so, I think we need to update our models and get consultation regarding any derived computations on that node status.

Reply:
The API is just a pass-through for the Node's status field. It's an explicit status, not a computed status from the API's perspective. You should get a new text string "disconnected" if the node is disconnected.

Reply:
Great. So the `status` of the `Node` will be `disconnected`. We'll need to update our model to reflect that (and track down if we have any derived computations based on that in the UI).